src/app/components/model_tree/model_tree_line.js

```diff
@@ -93,6 +93,10 @@ angular
                     file: {
                         on: 'icn-doc-on',
                         off: 'icn-doc'
+                    },
+                    group: {
+                        on: 'icn-filter',
+                        off: 'icn-filter'
```

Comment:
Using the same values here means they just won't change when the group is expanded.  There was no preexisting `icn-filter-on`.  This bring the most value for now.